test/functional/wallet_dump.py

```diff
@@ -202,5 +202,10 @@ def run_test(self):
         result = self.nodes[0].getaddressinfo(multisig_addr)
         assert result['ismine']
 
+        self.log.info('Check that wallet is flushed')
+        with self.nodes[0].assert_debug_log(['Flushing wallet.dat'], timeout=20):
```

Comment:
I think both can be done at the same time, but asserting the modification time increases requires a `time.sleep(1)`, since `os.path.getmtime` returns seconds. I don't like sleeps longer than 200 ms in tests, so I'll leave that for a follow-up ;)

Reply:
No need for a follow-up. Fast tests are nice too!